hnn_core/drives.py

```diff
@@ -40,7 +40,7 @@ def _get_target_properties(weights_ampa, weights_nmda, synaptic_delays,
     if location == 'distal' and 'L5_basket' in target_populations:
         raise ValueError('When adding a distal drive, synaptic weight cannot '
                          'be defined for the L5_basket cell type as this '
-                         'connection does not exist.')
+                         'connection does not exist. Please delete the L5_basket part from the code.')
```

Comment:
This right here produces a "flake8" error. flake8 is basically a program that enforces certain rules for how code should be styled. Here the line is too long so you should break this string into 2 lines.

You can [enable flake8](https://code.visualstudio.com/docs/python/linting) on VSCode so that it highlights areas that break [PEP8](https://peps.python.org/pep-0008/) style conventions

Reply:
I think there is a more precise and explicit way to write this error message. What about something like this?

```python
f"Due to physiological/anatomical constraints, a distal drive cannot target L5_basket cell types. L5_basket cell types must remain undefined by the user in all synaptic weights dictionaries for this drive ({drive_name})".
```

Reply:
> I think there is a more precise and explicit way to write this error message. What about something like this?
> 
> ```python
> f"Due to physiological/anatomical constraints, a distal drive cannot target L5_basket cell types. L5_basket cell types must remain undefined by the user in all synaptic weights dictionaries for this drive ({drive_name})".
> ```

Thank you for the suggestion. I will fix it!